torchvision/prototype/transforms/_misc.py

```diff
@@ -18,6 +18,8 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 class Lambda(Transform):
     def __init__(self, fn: Callable[[Any], Any], *types: Type):
+        # datumbox: the use of types here is BC breaking. Perhaps we should ensure the default behaviour when types
+        # is not passed aligns with the old behaviour.
```

Comment:
cc @vfdev-5 / @pmeier

Reply:
The default should be "transform everything", right? If yes, we can simply do

```py
if not types:
    types = (object,)
```

Reply:
See https://github.com/pytorch/vision/pull/6492/commits/a0b3fbb43ff6a7c06738cb9676260582ac7a3ec7

Reply:
Addressed by #6492